packages/hierarchies-react/src/presentation-hierarchies-react/stratakit/RenameAction.tsx

```diff
@@ -23,7 +37,7 @@ export const RenameAction = memo(function RenameAction() {
   }, [setIsRenaming]);
 
   if (!context?.onLabelChanged) {
-    return undefined;
+    return reserveSpace ? <Tree.ItemAction label={rename} onClick={handleClick} visible={false} disabled icon={renameSvg} /> : undefined;
```

Comment:
avoid duplicating `<Tree.ItemAction />` rendering

Reply:
I'd rather keep the reserve action and the actual action separate. Even if it has some code duplication, I think it keeps it cleaner in a way that the handling of undefined component is in one return instead of multiple as well as no need to extend current visible and disabled logic for the actual action.

Reply:
thanks, I can live with that